networkx/algorithms/core.py

```diff
@@ -491,17 +491,19 @@ def k_truss(G, k):
     ------
     NetworkXError
 
-      The k-truss is not defined for graphs with self loops or parallel edges
-      or directed graphs.
+      The k-truss is not defined for graphs with parallel edges, directed and multi graphs.
 
     Notes
     -----
     A k-clique is a (k-2)-truss and a k-truss is a (k+1)-core.
 
-    Not implemented for digraphs or graphs with parallel edges or self loops.
+    Not implemented for digraphs or graphs with parallel edges.
 
     Graph, node, and edge attributes are copied to the subgraph.
 
+    k_truss function uses copy() method to remove any self loops already present. Hence it
```

Comment:
Hmmm...  I don't think the `copy` method removes selfloops. Can you pin that down? Maybe making some tests would show the behavior you are documenting.

Reply:
Sure, will do!

Reply:
You're right! `copy` method, does not remove self loops. I will revert it to the original status. This issue needs to be dealt with separately on #6508 . Thanks a lot for the clarification @dschult :)